src/history/InferredQuorumUtils.cpp

```diff
@@ -24,8 +25,11 @@ checkQuorumIntersection(Config const& cfg)
     Config cfg2(cfg);
     cfg2.setNoListen();
     Application::pointer app = Application::create(clock, cfg2, false);
-    InferredQuorum iq = app->getHistoryManager().inferQuorum();
-    iq.checkQuorumIntersection(cfg2);
+    LOG(INFO) << "Checking last-heard quorum from herder";
+    app->start();
+    auto qic = QuorumIntersectionChecker::create(
```

Comment:
I modified it so that it defaults to use the herder's persistent local state, and can use any other ledger number K that you ask for, out of history, if you provide a `--history-ledger` command line value. There's little point specifying a variable N for the number of ledgers to scan before K looking for qsets, as it'll be quite arbitrary whether there are "enough" there to fill in the qsets you need; I just left it hard-coded to N=100 checkpoints, which is 9 hours of ledgers, quick enough to download but long enough that you're not likely to get any more qsets by downloading any more.